Source/Diagnostics/ReducedDiags/ColliderRelevant.cpp

```diff
@@ -0,0 +1,559 @@
+/* Copyright 2023 The WarpX Community
+ *
+ * This file is part of WarpX.
+ *
+ * Authors: Yinjian Zhao, Arianna Formenti
```

Comment:
Mostly you, you can credit Yin for his earlier reduced diags here if sufficiently similar
```suggestion
 * Authors: Arianna Formenti, Yinjian Zhao
```